beebop/app.py

```diff
@@ -15,16 +34,107 @@ def response_success(data):
     return response
 
 
+def response_failure(error):
+    response = {
+        "status": "failure",
+        "errors": [error],
+        "data": []
+    }
+    return response
+
+
+def check_connection(redis):
+    try:
+        redis.ping()
+        return True
```

Comment:
No need to return `True` here (and later you assume you _might_ return `False` which you never do!